webcompat/static/js/lib/issues.js

```diff
@@ -436,6 +438,24 @@ issues.MainView = Backbone.View.extend({
         wcEvents.trigger('flash:error', {message: msg, timeout: 4000});
       }
     });
+    var i;
+    for (i = 2; i <= this.lastPageNumber; i++) {
```

Comment:
You can move `i` initialization inside the for loop => `for (var i = 2....`

Can you also add a comment why you're starting with 2?